src/traintest.jl

```diff
@@ -4,14 +4,14 @@ function train(dre::DRE, eeg_in, eeg_out, ps, st; n_epochs=1, lr=0.01, batch_siz
     @info "Input data is a $(typeof(eeg_in)) and $(typeof(eeg_out)) with size $(size(eeg_in)),$(size(eeg_out))"
     # conv layer expects input of shape (time, channel, epoch)
     opt_state = create_optimiser(ps, lr)
-    p = Progress(n_epochs) # TODO:  replace ProgressMeter.jl with ProgressLogging.jl
+    #p = Progress(n_epochs) # TODO:  replace ProgressMeter.jl with ProgressLogging.jl
```

Comment:
can be remoevd

Reply:
Actually after you mentioned, I'm looking at ways of how we can add a parameter called verbose and based on the level show either only the top level progress or progress for each epoch. Makes sense to do it this was since currently, say we have 1000 epochs, the screen will be filled up with 1000 progress bars one after the other

Reply:
I think this might be premature optimization, I would move on and think about it only later.

You could always do it manually, by progressing the progressmeter by steps (see their docu)